The name 'GraphicsDeviceType' does not exist in the current context

### Plugin Version or Commit ID

v0.14.0

### Unity Version

2021.3.33f1c1

### Your Host OS

Windows10

### Target Platform

Android

### Target Device

_No response_

### [Windows Only] Visual Studio C++ and Windows SDK Version

_No response_

### [Linux Only] GCC/G++ and GLIBC Version

_No response_

### [Android Only] Android Build Tools and NDK Version

_No response_

### [iOS Only] XCode Version

_No response_

### Build Command

no

### Bug Description

it's all right on unity editor, but when i build on Android, it has an error (Assets\MediaPipeUnity\Samples\Common\Scripts\GraphRunner.cs(52,48): error CS0103: The name 'GraphicsDeviceType' does not exist in the current context).i use the Pose Tracking demo.

### Steps to Reproduce the Bug

File-->Build Settings-->Android-->Build

### Log

Assets\MediaPipeUnity\Samples\Common\Scripts\GraphRunner.cs(52,48): error CS0103: The name 'GraphicsDeviceType' does not exist in the current context

### Screenshot/Video

_No response_

### Additional Context

_No response_

--- a/Assets/MediaPipeUnity/Samples/Common/Scripts/GraphRunner.cs
+++ b/Assets/MediaPipeUnity/Samples/Common/Scripts/GraphRunner.cs
@@ -11,6 +11,10 @@ using System.Linq;
 using System.Threading.Tasks;
 using UnityEngine;
 
+#if UNITY_ANDROID
+using UnityEngine.Rendering;
+#endif
+
 using Stopwatch = System.Diagnostics.Stopwatch;
 
 namespace Mediapipe.Unity.Sample

